src/App.jsx

```diff
@@ -14,33 +16,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_GOODS_ALPHABET = 'Alphabet';
+const SORT_GOODS_LENGTH = 'Length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, sortFild, reversed) {
+  const preparedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  preparedGoods.sort((good1, good2) => {
+    switch (sortFild) {
+      case SORT_GOODS_ALPHABET:
+        return good1.localeCompare(good2);
+      case SORT_GOODS_LENGTH:
+        return good1.length - good2.length;
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+      default:
+        return 0;
+    }
+  });
+
+  if (reversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
+export const App = () => {
+  const [sortFild, setSortFild] = useState('');
+  const [reversed, setReversed] = useState(false);
+  const visibleGoods = getPreparedGoods(goodsFromServer, sortFild, reversed);
+  const isChoice = sortFild || reversed;
```

Comment:
1. It good practice to add empty line to separete blocks of code 
2. Let's rename `isChoice` to something more descriptive


```suggestion
  const [sortFild, setSortFild] = useState('');
  const [reversed, setReversed] = useState(false);
  
  const visibleGoods = getPreparedGoods(goodsFromServer, sortFild, reversed);
  
  const isChoice = sortFild || reversed;
```